hyp3_testing/helpers.py

```diff
@@ -118,9 +118,11 @@ def archive_tifs(product_name, product_urls, directory, keep=False):
     if not product_dir.exists():
         product_dir.mkdir(parents=True)
         [download_file(url, product_dir) for url in product_urls]
-    tif_paths = sorted(product_dir.glob('*.tif'))
+    tif_paths = product_dir.glob('*.tif')
+    browse_path = product_dir.glob('*BROWSE.png')
```

Comment:
You might not need to add this at all since we're globbing for the browses later anyways. Plus this could break `archive_tifs` for the other tests